extensions/smallrye-graphql/runtime/src/main/java/io/quarkus/smallrye/graphql/runtime/SmallRyeGraphQLExecutionHandler.java

```diff
@@ -297,7 +297,8 @@ private static String getCharset(String mimeType) {
     private boolean isValidAcceptRequest(String mimeType) {
         // At this point we only accept two
```

Comment:
Change looks good to fix the tests but this comment looks outdated. I wonder if we should explain a bit more and put the two valid ones first and the last one with a comment `// deprecated content-type supported for compatibility`.

Reply:
Ah, that's a very good observation, thanks. Fixed